frontend/src/component/ContentListItem.styl

```diff
@@ -7,8 +7,10 @@
   display flex
   justify-content space-between
   color fontColor
-  height 60px
+  height 65px
   align-items center
+  & > *
+    margin 0 8px
```

Comment:
Maybe I do not understand the fix/issue, but it seems that it just add the left-right margins to all lists, like for notification wall:

![Capture d’écran de 2022-10-27 09-31-41](https://user-images.githubusercontent.com/35422692/198219624-64c5bc7f-3590-4fb3-b268-73efd5c9b821.png)

Shoudn't the fix be about #5802 (that could be fix with a min-height) ?

Reply:
The margins purpose is to keep a blank space between each field because one of the problems with the readability were also caused by horizontal spacing, no just the vertical one. I should indeed used min-height (I'm fixing this right now) but the whole goal was to bring more readability in the favorites, not just linked to the vertical spacing

### How it looks with the fix
![image](https://user-images.githubusercontent.com/42064302/198224813-f43a1e55-87fb-4265-b65e-0945997ff45b.png)

### How it looked before
![image](https://user-images.githubusercontent.com/42064302/198225218-c3bf4c3b-d390-451f-9ce2-32f43d149fb1.png)

As you can see, the different fields were too close to each other, resulting in poor readability. This fix brings a less dense content display for the favorites.

I also checked for the other usages of the content list and fortunately it give also a lighter interface.

Reply:
However, this fix could also fix #5802 as it solves this problem too.

Reply:
You are right, I applied the fix in the wrong place. I'm still not 100% convinced because it generate different spacing inside and outside the elements (orange in the screenshot), but I think it was already the case, so not a strong opinion.
![Capture d’écran de 2022-10-27 10-10-51](https://user-images.githubusercontent.com/35422692/198229413-9391fe57-098e-4a2b-9204-f1f2bb347462.png)

I suggest use the standardSpacing

```suggestion
    margin 0 standardSpacing
```